packages/react/src/components/TypingUsers/index.js

```diff
@@ -17,32 +17,84 @@ export default function TypingUsers() {
   }, [RCInstance, setTypingUsers, currentUserName]);
 
   const typingStatusMessage = useMemo(() => {
-    if (typingUsers.length === 0) return '';
-    if (typingUsers.length === 1)
-      return (
-        <span>
-          <b>{typingUsers[0]}</b>
-          {' is typing...'}
-        </span>
-      );
-    if (typingUsers.length === 2)
+    if (!typingUsers.length) return '';
+
+    if (typingUsers.length) {
+      const otherTypingUsers = typingUsers.filter((user) => user !== currentUserName);
+
+      if (typingUsers.length === 1) {
```

Comment:
When you have already filtered typingUsers, and stored the list in otherTypingUsers, the later variable should be used afterwards. However, I wonder why we need to refilter, when it is already done in useEffect. The issue might be somewhere else if its happening.

Reply:
Hi @abhinavkrin . Actually, the previous approach wasn't giving the correct output, which should be like - **not to show that the "current user" is typing, and show it only when others are typing**.

So, I had to resolve to use this manual approach of using if-else which covers all the cases. It works fine btw.

Reply:
can i work on this and resolve this?